src/Areas/Server/Options/ServiceStartOptions.cs

```diff
@@ -26,10 +26,10 @@ public class ServiceStartOptions
 
     /// <summary>
     /// Gets or sets the mode mode for the server.
-    /// When null, services are exposed individually.
+    /// Defaults to 'namespace' mode which exposes one tool per service namespace.
     /// </summary>
     [JsonPropertyName("mode")]
-    public string? Mode { get; set; } = null;
+    public string? Mode { get; set; } = ModeTypes.NamespaceProxy;
```

Comment:
We also need to ensure we have a new `ModeTypes.All` which allows the MCP server to launch with all tools enabled.  It should support startup with command like `azmcp server start --mode all`.

Reply:
> --mode null

Yes the "--mode null" above needs to be replaced with "--mode all"